_posts/2018-07-25-cockpit-173.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Cockpit 173
+author: pitti
+date: 2018-07-25
+tags: cockpit linux
+slug: cockpit-173
+category: release
+summary: Cockpit with on-demand VDO install and VM disk creation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly.
+Here are the release notes from version 173.
+
+### Storage: Offer installation of VDO
+
+The Storage page now shows an "Install VDO support" link if the
+[VDO](https://github.com/dm-vdo/vdo) package is not installed. This is similar
+to the on-demand installation of the NFS client packages in
+[Cockpit 169](https://cockpit-project.org/blog/cockpit-169.html).
+
+![Install VDO on demand](/images/vdo-install-on-demand.png)
+
+### Machines: Add disks to virtual machine
```

Comment:
I'd write either "to a virtual machine" or (better) "to virtual machines".

Reply:
Thanks! Fixed.